src/swarm/neo/client/mixins/ClientCore.d

```diff
@@ -161,6 +161,7 @@ template ClientCore ( )
         this.connections = new ConnectionSet(cred, this.outer.epoll,
             settings.conn_notifier, settings.auto_connect);
         this.request_resources = settings.request_resources;
+        this.enableSocketNoDelay();
```

Comment:
This case isn't mentioned in the commit message. What's the reasoning for doing it here as well?

Reply:
It has to be done before the `Connections` inside `ConnectionSet` are created - since it's passed via constructor of every `Connection` object. Was this not clear enough - do you prefer more detailed explanation?

> In case the NeoConfig enabled constructor is used, the TCP_NODELAY
can't be set downstream, because it needs to be set before the nodes
are added [...]

Reply:
It's just that the commit message says "In case the NeoConfig enabled constructor is used". That implies just the 2nd ctor, or?

Reply:
Ah, you're right. I'll reword.

Reply:
Reworded.